app/jsx/components/HeaderComp.jsx

```diff
@@ -25,29 +26,31 @@ class HeaderComp extends React.Component {
             <button className="c-header__deposit-button">Deposit/Publish</button>
           </div>
         </div>
-        { this.props.level && this.renderLocalHeader(this.props.level) }
+        { this.props.level && this.props.unit_id != 'root' && this.renderLocalHeader() }
       </div>
     )
   }
 
   renderLocalHeader() {
+    // onChange={this.handleCampusChange}
```

Comment:
Still need to build this handler. I also would like to put top level campuses into an object to be passed instead of manually displaying them here.